src/resources/postcss/tribe-events-full.pcss

```diff
@@ -194,11 +194,13 @@ a.tribe-button,
 =============================================*/
 #tribe-events,
 #tribe-events-pg-template,
-.tribe-events-pg-template {
+.tribe-events-pg-template,
+#top.theme-enfold #tribe-events-pg-template {
 	padding: 40px 20px 20px;
 }
 
 #tribe-events-pg-template,
+#top.theme-enfold #tribe-events-pg-template,
```

Comment:
Curious why we're changing the page template here for enfold users. Isn't the goal here to only reskin the organizer block?

Reply:
@EugeneKyale any comment on this?

Reply:
@PaulHughes01 I've removed the override targeting Enfold.